app/views/admin/editions/_style_guidance.html.erb

```diff
@@ -0,0 +1,61 @@
+<h2 class="govuk-heading-l">Styling</h2>
+
+<p class="govuk-body">For styles, see the <%= link_to "style guide", "https://www.gov.uk/guidance/style-guide", class: "govuk-link" %>
+
+<p class="govuk-body">For details, see the <%= link_to "guideline on using plain English", "https://www.gov.uk/guidance/content-design/writing-for-gov-uk#plain-english"%>
```

Comment:
This link is missing `class: "govuk-link"` – so it's ended up being a slightly different shade of blue 🙈 

```suggestion
<p class="govuk-body">For details, see the <%= link_to "guideline on using plain English", "https://www.gov.uk/guidance/content-design/writing-for-gov-uk#plain-english", class: "govuk-link" %>
```

Reply:
Oh yeah, class added 👍